packages/algob/src/lib/deployer.ts

```diff
@@ -137,4 +146,12 @@ export class AlgobDeployerReadOnlyImpl implements AlgobDeployer {
   isDefined (name: string): boolean {
     return this._internal.isDefined(name);
   }
+
+  get algodClient() {
+    return this._internal.algodClient
+  }
+
+  async waitForConfirmation (txId: string): Promise<algosdk.ConfirmedTxInfo> {
+    return this._internal.waitForConfirmation(txId)
+  }
```

Comment:
There should be some inheritance to avoid code duplication.

Reply:
I wanted to explicitly not inherit in this class so that compiler would fail when interface is not implemented.

Reply:
We can have some complex inheritance where only a part of it is inherited but I don't think it's worth it.